src/pages/AboutPage/AboutPage.Styles.js

```diff
@@ -0,0 +1,21 @@
+import styled from 'styled-components';
+
+export const DivContainer = styled.div`
```

Comment:
you are not supposed to prefix every item with it'sunderlying element type. That's the point: you have a power to express those with abstract names. Wrapper, PageWrapper, MainWrapper... ContainerOfTheWorld... kidding. The point is that noone needs to know that it was the p tag that was used under the hood.

it is applicable to all other places